target/linux/ramips/mt7620/profiles/zbtlink.mk

```diff
@@ -0,0 +1,17 @@
+#
```

Comment:
The default profile includes USB packages:
  PACKAGES:= kmod-usb-core kmod-usb2 kmod-usb-ohci kmod-usb-ledtrig-usbport

WE2026 doesn't have USB port, so we need to get rid of the USB stuff and create new profile without USB support.
  PACKAGES:=